src/Adapter/Builder/FtpAdapterDefinitionBuilder.php

```diff
@@ -60,6 +60,9 @@ protected function configureOptions(OptionsResolver $resolver)
 
         $resolver->setDefault('ignore_passive_address', null);
         $resolver->setAllowedTypes('ignore_passive_address', ['null', 'bool']);
+
+        $resolver->setDefault('utf8', false);
```

Comment:
I didn't have a deep look but I wasn't able to find the default value in Flysystem for this option. Are you use it's false? If it's not, this is a BC break.

Reply:
Well, in `League\Flysystem\Adapter\Ftp` the default value is set with `protected $utf8 = false;`.